components/org.wso2.carbon.identity.saml.common.util/src/main/java/org/wso2/carbon/identity/saml/common/util/SAMLInitializer.java

```diff
@@ -75,6 +75,9 @@ public static void doBootstrap() throws InitializationException {
             GlobalAlgorithmRegistryInitializer globalAlgorithmRegistryInitializer = new GlobalAlgorithmRegistryInitializer();
             globalAlgorithmRegistryInitializer.init();
 
+            org.opensaml.soap.config.XMLObjectProviderInitializer soapXMLObjectProviderInitializer = new org.opensaml.soap.config.XMLObjectProviderInitializer();
```

Comment:
Please check the line length. It exceeds 120 characters

Reply:
fixed